src/message.ts

```diff
@@ -7,10 +7,14 @@ export class Message {
     private message: broker.JsMsg;
     private connection: Memphis;
     private cgName: string;
-    constructor(message: broker.JsMsg, connection: Memphis, cgName: string) {
+    private stationName: string;
+    private internal_station: string;
+    constructor(message: broker.JsMsg, connection: Memphis, cgName: string, stationName: string) {
         this.message = message;
         this.connection = connection;
         this.cgName = cgName;
+        this.stationName = stationName;
```

Comment:
you can pass the internal name directly rather than do this strings operations for each message